src/App.tsx

```diff
@@ -144,12 +144,12 @@ function App({ walletReady }: AppProps) {
     }
     identifyFlow();
     eventSubscription.current = AppState.addEventListener('change', handleAppStateChange);
+    initWalletConnectListeners();
```

Comment:
any idea if we need to wait on any of the proomises that wrapped this for any reason?

Reply:
Push notifications. Just fixed it on a separate commit